### ISSUE TITLE
wmk perf --module: --module flag expects class name not module path; unclear error when wrong format used

### ISSUE THREAD

**Issue by DingmaomaoBJTU** [microsoft/winml-cli#195]
## Summary

\`wmk perf --module\` expects a **PyTorch module class name** (e.g., \`ResNetStage\`, \`BertAttention\`), not a dotted module path. When an incorrect format is passed, the error message does not guide the user toward the correct usage. Additionally, CLIP's \`CLIPEncoderLayer\` fails because its \`forward()\` requires \`attention_mask\` and \`causal_attention_mask\` arguments that the random input generator does not supply.

## Repro

\`\`\`bash
# Wrong: module path (fails silently or with confusing error)
wmk perf -m openai/clip-vit-base-patch32 --module vision_model

# Correct: class name
wmk perf -m microsoft/resnet-50 --module ResNetStage

# Still fails: CLIP class name (forward signature issue)
wmk perf -m openai/clip-vit-base-patch32 --module CLIPEncoderLayer
# Error: CLIPEncoderLayer.forward() missing attention_mask and causal_attention_mask
\`\`\`

## Issues

1. **Documentation gap**: \`wmk perf --help\` states \`HF module class name\` but the distinction from module path is not obvious to users
2. **CLIP forward signature**: \`CLIPEncoderLayer.forward()\` requires extra keyword arguments (\`attention_mask\`, \`causal_attention_mask\`) that \`generate_random_inputs()\` does not generate, causing all 12 CLIP layer benchmarks to fail

## Acceptance Criteria

- [ ] \`wmk perf --module\` help text clearly states class name is required with an example
- [ ] Passing a dotted path (e.g., \`encoder.layers.0\`) produces a clear error: "did you mean class name?"
- [ ] Random input generator handles attention mask shapes for CLIP/transformer modules, or a \`--module-kwargs\` escape hatch is provided

**Comment by DingmaomaoBJTU** [comment]
**Triage summary**

- **Component**: Perf
- **Priority**: P2
- **Owner(s)**: @DingmaomaoBJTU

wmk perf --module expects a class name but the help text doesn't make this clear, and CLIP's CLIPEncoderLayer fails due to missing required kwargs in random input generation; improve help text and handle attention mask shapes for transformer modules.

**Comment by xieofxie** [comment]
clip issue fixed in https://github.com/microsoft/WinML-ModelKit/pull/586
